auth-api/src/auth_api/resources/v1/org_products.py

```diff
@@ -65,3 +65,22 @@ def post_org_product_subscription(org_id):
     except BusinessException as exception:
         response, status = {'code': exception.code, 'message': exception.message}, exception.status_code
     return response, status
+
+
+@bp.route('', methods=['PATCH'])
+@cross_origin(origins='*')
+@TRACER.trace()
+@_jwt.has_one_of_roles([Role.PUBLIC_USER.value])
```

Comment:
Do staff need this functionality too? They don't have `PUBLIC_USER`

Reply:
Staff will be using the dashboard to transition the status of a task. This is meant to be be triggered from a public user when they resubmit an application specifically Qualified Supplier in this case

Reply:
auth check in there too, good stuff